classes/Mission.php

```diff
@@ -320,4 +363,47 @@ public static function maxMissionRank(int $player_rank): int {
         }
         return $max_mission_rank;
     }
-}
\ No newline at end of file
+
+    public static function processRewards(Mission $mission, User $player, System $system): string {
+        if (count($mission->rewards) > 0) {
+            try {
+                $reward_text = "";
+                // load player inventory
+                $player->getInventory();
+
+                // get items from DB
+                $reward_keys = [];
+                foreach ($mission->rewards as $item) {
```

Comment:
If these are specifically item IDs, could we name this `$mission->reward_item_ids`?

Reply:
Resolved in commit.